packages/web-react/src/components/FileUploader/README.md

```diff
@@ -347,30 +365,31 @@ and a collection of props selected above for easier implementation. Other props
-| `linkText`            | `string`                                | -       | ✕        | Link text in input in Drop zone               |
-| `attachmentComponent` | `string`                                | -       | ✔        | A component for rendering a single attachment |
-| `maxFileSize`         | `number`                                | 1000000 | ✕        | The maximum size of the uploaded file         |
-| `maxUploadedFiles`    | `number`                                | 10      | ✕        | Maximum file upload queue size                |
-| `isMultiple`          | `boolean`                               | -       | ✕        | When multiple files can be selected at once   |
-| `isDisabled`          | `boolean`                               | -       | ✕        | Whether is field disabled                     |
-| `isRequired`          | `boolean`                               | -       | ✕        | Whether is field required                     |
-| `onInputError`        | `FileUploaderErrorCallbackType`         | -       | ✕        | Callback on error condition                   |
-| `onChange`            | `(fileQueue: FileQueueMapType) => void` | -       | ✕        | Callback on change in fileQueue               |
-| `validationState`     | `ValidationState`                       | -       | ✕        | Validation state                              |
-| `validationText`      | `string`                                | -       | ✕        | Validation status text                        |
-| `UNSAFE_className`    | `string`                                | -       | ✕        | UncontrolledFileUploader custom class name    |
-| `UNSAFE_style`        | `CSSProperties`                         | -       | ✕        | UncontrolledFileUploader custom style         |
+| Prop name             | Type                                    | Default | Required | Description                                                                    |
+| --------------------- | --------------------------------------- | ------- | -------- | ------------------------------------------------------------------------------ |
+| `id`                  | `string`                                | -       | ✔        | UncontrolledFileUploader id                                                    |
+| `inputId`             | `string`                                | -       | ✔        | FileUploaderInput id                                                           |
+| `inputLabel`          | `string`                                | -       | ✔        | FileUploaderInput label                                                        |
+| `inputName`           | `string`                                | -       | ✔        | FileUploaderInput name                                                         |
+| `inputProps`          | `Partial<FileUploaderInputBaseProps>`   | -       | ✕        | Rest of FileUploaderInput props                                                |
```

Comment:
Oh, now I noticed 🙈. When a props is not required, it has a default value (`''`, `null`, `false`, … see the [`Button` `README`](https://github.com/lmc-eu/spirit-design-system/blob/main/packages/web-react/src/components/Button/README.md#available-props), for example) that needs to be listed here in the API table.

Now when we are on it, if you have time, would please fix it? 🙏🏻  But we can do it later, no pressure.

cc @literat

Reply:
But that's wrong. Not always is default value `null` or `false`. Depends on type. For example, the unspecified value for children is not `null` but `undefined`